schemas/file.yml

```diff
@@ -225,3 +225,17 @@
 
       short: A fork is additional data associated with a filesystem object.
       example: Zone.Identifer
+
+    - name: origin_referrer_url
+      level: extended
+      type: keyword
+      ignore_above: 8192
```

Comment:
This is a fairly large ignore_above value, do you expect that there will be data this size? Have you looked into the performance impact?

Reply:
>do you expect that there will be data this size? 
Yes. When defining the URL size, I did some research and came across a discussion from a past ECS thread about URL sizes, which I used as a reference. There was a discussion that the URL field should be at least 4096 bytes, and ideally 8192 bytes based on the actual URL log data, so I followed that suggestion. In addition, after discussing it within the team, we concluded that since the typical maximum URL size accepted by servers is 8KB, 8KB seemed like the most reasonable choice.